app/models/api_key.rb

```diff
@@ -0,0 +1,11 @@
+class ApiKey < ApplicationRecord
+  before_create :generate_access_token
+
+  private
+
+  def generate_access_token
+    begin
+      self.access_token = SecureRandom.hex
```

Comment:
`hex` only allows 0-9 and a-f. You'll have a wider character set with Base64: `SecureRandom.urlsafe_base64(length)`

Reply:
No longer required  - using basic auth